doc.go

```diff
@@ -22,7 +22,9 @@
 //
 // Abstract
 //
-// [dosa][dosa-spec] is a library that provides a distributed
+// DOSA open source is not ready for prime time yet. We will announce when it's ready.
```

Comment:
We have a build tool to make go doc based on readme. Readme has been updated to those statement.